lib/client-assets.php

```diff
@@ -371,5 +371,12 @@ function gutenberg_scripts_and_styles( $hook ) {
 		array( 'wp-components', 'wp-blocks' ),
 		filemtime( gutenberg_dir_path() . 'editor/build/style.css' )
 	);
+
+	global $hook_suffix;
+	/**
+	 * Hook to allow extending of gutenberg editor in plugin/themes.
+	 */
+	do_action( 'block_enqueue_script', $hook_suffix );
```

Comment:
@westonruter i tried to replicate the `admin_enqueue_script` behaviour, which passes the $hook_sufix when calling the action.